pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -686,6 +687,10 @@ def _check_and_init_precision(self) -> PrecisionPlugin:
                     return ShardedNativeMixedPrecisionPlugin(self._precision_flag, device)
                 if isinstance(self.strategy, DDPFullyShardedStrategy):
                     return FullyShardedNativeMixedPrecisionPlugin(self._precision_flag, device)
+                if isinstance(self.strategy, DDPFullyShardedNativeStrategy):
+                    raise MisconfigurationException(
+                        "`DDPFullyShardedNativeStrategy` currently doesn't support Mixed Precision."
+                    )
```

Comment:
This misconfig doesn't only apply to amp native right? apex is not supported either?
Can we move this to L682?

Reply:
Makes sense.